### ISSUE TITLE
Tree list items, refresh and more button is not accessible via keyboard tab key

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/vscode#180111]
**GitHubTags:**#A11yMAS;#A11yTCS;#A11ySev1;#Keyboard;#Desktopweb;#WCAG2.1.1;#A11yAuto;#Win10;#AILimited;#TabStop;#FTP;#A11y_VSCode;#CAITestPass_May2020;#Visual Studio Code;

### Environment Details:
#URL: https://ms.portal.azure.com/#home
Chromium edge: Version 95.0.1020.44 (Official build) (64-bit)

### Repro Steps:

1. Open URL https://ms.portal.azure.com/
2. Navigate to the cloud shell button on the top menu bar and select it.
3. Navigate to "Open Editor" button of "CloudShell-PowerShell Editor" window and select it
4. press tab to move to the editor and then, press tab and ctrl+M
5. use shift +tab focus goes on the entire tree list but not individual items

### Actual:
Tree views, refresh and more button is not accessible via keyboard tab key. on shift+tab to the tree list the arrow keys do not move to the items.

### Expected Result:
Tree views, refresh and more button should accessible via keyboard tab and arrow keys

### User Impact:
keyboard users will not be able to access the controls.

### Attachment:
![143035591-e2037a87-9bc0-40ea-b840-adcc7b781bbb](https://user-images.githubusercontent.com/104351566/232455805-c2ff2cd9-3377-41c0-a7bf-830d16b2415c.gif)

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by BhanuPrakashGit** [comment]
@alexr00 Once you will hit the URL: https://insiders.vscode.dev/, you'll get the similar screen.
Below is the screen you'll get.

![image](https://user-images.githubusercontent.com/104351566/235873772-07b375e9-ae69-4cfd-a584-cefbf8ef6b13.png)

**Comment by alexr00** [comment]
@BhanuPrakashGit can you direct me to which tree items, refresh, and more buttons are not accessible via keyboard tab?

**Comment by BhanuPrakashGit** [comment]
@alexr00 in this VS code URL, we are able to access every control via keyboard tab.

**Comment by alexr00** [comment]
@BhanuPrakashGit if everything is accessible at https://insiders.vscode.dev/ then this is not an issue with VS Code.

**Comment by BhanuPrakashGit** [comment]
@alexr00 yes there is no issue with VS code. We have logged bug for the URL: https://ms.portal.azure.com/, here that issue is repro'ing.

**Comment by fsteffi** [comment]
HI @alexr00 can you please open this bug

**Comment by alexr00** [comment]
The bug doesn't reproduce in in VS Code, so this bug should remain closed.

**Comment by fsteffi** [comment]
#closed;
